site/docs/guidelines/checkboxs-radiobuttons-toggles.mdx

```diff
@@ -0,0 +1,152 @@
+        <Checkbox id="v-checkbox2" name="v-checkbox2" label="Arts and museums" checked={checkedItems["v-checkbox2"]} onChange={onChange} />
+        <Checkbox id="v-checkbox3" name="v-checkbox3" label="Sports and outdoor activities" checked={checkedItems["v-checkbox3"]} onChange={onChange} />
+        <Checkbox id="v-checkbox4" name="v-checkbox4" label="Cooking" checked={checkedItems["v-checkbox4"]} onChange={onChange} />
+      </SelectionGroup>
+    );
+  }}
+</Playground>
+
+<br />
+<b>HDS recommends using checkboxes when the user can select multiple options that are not excluding other options. </b>
+
+## When to use toggles or switches?
+<Link href="../components/toggle">HDS toggle component</Link>
+<br /><br />
+Toggle switches are digital on/off switches. They prompt users to choose between two mutually exclusive options and always have a default value. Toggle buttons have been inspired by real-world physical switches (e.g. a light switch). Therefore users expect something to happen immediately when the toggle button is activated. Toggle buttons should do three things — change states, show the current state, and reveal unselected options. If your toggle button doesn’t do all these things, it has poor usability. Toggles are best used for switching between two opposing states of system functionality and preferences. They are similar to radio buttons or checkboxes in that they allow users to select from only two options.
+<br /><br />
+<b>Example use:</b> Switching between themes.
+
+<Playground>
+  {() => {
+    const [hasCustomTheme, setHasCustomTheme] = React.useState(false);
+    const customTheme = {
+      title: 'Primary theme',
+      label: 'Use primary theme',
+      baseStyles: {
+        ['--label-color-default']: 'var(--color-coat-of-arms)'
+      },
+      titleStyles: {
+        color: 'var(--color-coat-of-arms)'
+      },
+      toggle: {
+        '--toggle-button-color': 'var(--color-coat-of-arms)',
+        '--toggle-button-hover-color': 'var(--color-coat-of-arms)',
+      }
+    };
+    return (
+      <div style={(hasCustomTheme ? customTheme.baseStyles : {})}>
+        <h2
+          style={(hasCustomTheme ? customTheme.titleStyles : undefined)}>
+          {(hasCustomTheme ? customTheme.title : 'Default theme')}</h2>
+        <ToggleButton id="toggle-button-1"
+                      label={customTheme.label}
+                      checked={hasCustomTheme}
+                      theme={(hasCustomTheme ? customTheme.toggle : {})}
+                      onChange={() => setHasCustomTheme(!hasCustomTheme)} />
+        <br />
+      </div>
+    );
+  }}
+</Playground>
+
+
+
+## Sources and further reading
+Note that these articles might include guides that are not in line with HDS guidelines.
+
+
+<Link href="https://www.nngroup.com/articles/checkboxes-vs-radio-buttons/" external>Checkboxes vs. Radio Buttons, N&N Group</Link>
```

Comment:
These links should be put into a list to maintain consistency between other pages.